CMakeLists-implied-options.txt

```diff
@@ -87,6 +87,7 @@ if(LWS_WITH_DISTRO_RECOMMENDED)
 	set(LWS_WITH_LIBUV 1)				# libuv
 	set(LWS_WITH_LIBEV 1)				# libev
 	set(LWS_WITH_LIBEVENT 1)			# libevent
+	set(LWS_WITH_SDEVENT 0)				# sd-event
```

Comment:
Why is this disabled by default?

Reply:
I believe systemd is not as portable (it will probably never run on Windows), so I disable it by default.

Reply:
Make sense.